### PR TITLE
Fix idle backup seeding

### PR DESCRIPTION
**Pull request by martinfurmanski**
changelog: [causal clustering] Seeding a new member of an idle cluster using a backup from that idle cluster could fail.

### CODE REVIEW

File: enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/state/snapshot/CoreStateDownloader.java [hunk 1]
```diff
@@ -66,6 +66,8 @@ public CoreStateDownloader( FileSystemAbstraction fs, LocalDatabase localDatabas
     public synchronized void downloadSnapshot( MemberId source, CoreState coreState ) throws StoreCopyFailedException
     {
         // TODO: Think about recovery scenarios.
+        StoreCopyProcess storeCopyProcess = new StoreCopyProcess( fs, localDatabase, copiedStoreRecovery, log );
+
```
Comment (davidegrohmann) [inline enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/state/snapshot/CoreStateDownloader.java]: field?

File: enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/helpers/DataCreator.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        for ( int i = 0; i < numberOfNodes; i++ )
+        {
+            last = cluster.coreTx( ( db, tx ) ->
+            {
+                db.createNode();
+                tx.success();
+            } );
+        }
+        return last;
+    }
```
Comment (chrisvest) [inline enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/helpers/DataCreator.java:39]: There is already a class called `SampleData` that has a similar method.

**Comment by chrisvest** [comment]
The kernel changes look good.

File: enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/helpers/DataCreator.java (hunk 2 above)
Comment (martinfurmanski) [reply enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/helpers/DataCreator.java:39]: I'll take a look at that separately

**Pull request merged**